app/scripts/actions/index.js

```diff
@@ -155,6 +155,15 @@ export const deleteCollection = (name, version) => ({
   }
 });
 
+export const recoverCollection = (name, version) => ({
```

Comment:
Does there need to be an `action` property on this object? I just notice that there is one for `recoverGranule`

Reply:
Good call. I'm going to add it because there is a possibility of future functionality hitting this same endpoint for different actions.

Reply:
https://github.com/nasa/cumulus-dashboard/pull/569/commits/f73ab300344d599e3f86325cdc989f627a6e5206